apps/demos/Demos/Chat/MessageEditing/description.md

```diff
@@ -1,8 +1,10 @@
-The Chat component allows users to edit and delete sent messages.
+The Chat component allows to edit and delete messages sent by the current user.
 
-Use a [data source](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) if you want users to edit and delete messages. Implement custom insert, update, and delete operations. Once you configured these operations, enable [editing](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/).
+Use a [data source](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) if you want users to edit and delete messages. Implement [custom CRUD operations](/Documentation/ApiReference/Data_Layer/CustomStore/). Once you configured these operations, enable [editing](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/).
```

Comment:
```suggestion
Use a [data source](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) to allow users to edit and delete messages. Implement [custom CRUD operations](/Documentation/ApiReference/Data_Layer/CustomStore/). Once you configured these operations, enable [editing](/Documentation/ApiReference/UI_Components/dxChat/Configuration/editing/).
```
How does the second sentence in this paragraph relate to the first? Why do they need to implement custom CRUD operations? Be specific.